packages/base-types/src/version/settings.ts

```diff
@@ -40,10 +44,18 @@ export const defaultSettings = <Prompt>({
   session = { type: SessionType.RESTART },
   defaultCanvasNodeVisibility = null,
   defaultCarouselLayout = null,
+
+  globalNoMatch,
+  globalNoReply,
+  globalNoReplyDelay,
```

Comment:
didn't set the default values to null here, because we will need to diff between `undefined` and `null` in the future, to know if they are empty or untouched values.